### ISSUE TITLE
A formant processor

### ISSUE THREAD

**Issue by unfa** [calf-studio-gear/calf#124]

**Comment by harryhaaren** [comment]
"Formats" are basically specific settings for filters. JOS writes a detailed analysis of formant sounds here: https://ccrma.stanford.edu/~jos/fp/Formant_Filtering_Example.html

So *achieving* formant filtering is not difficult. IIUC, what you are looking for is a plugin that is capable of easily providing "musical" control over a bank of formant-filters, not "a formant processor" (aka, biquad filter).

The Propellorheads also do a great write-up of format-manipulation (with audio samples!)
https://www.propellerheads.se/blog/thor-demystified-17-filters-pt-5-formant-filters

I agree that linux-audio doesn't have one of these - and would be cool to have :)

**Comment by unfa** [comment]
Yes, I wondered if a vocoder in combination with a pitchshifter could be used as a formant shifter - but I guess that's produce a lot of bad side effects.

A dedicated effect plugin that could analyse and manipulate formant information in realtime, possibly with a sidechain input would be a really nice addition to the FLOSS music production arsenal.
